src/refda-stdio/main.c

```diff
@@ -163,6 +163,7 @@ static int stdin_recv(cace_ari_list_t data, cace_amm_msg_if_metadata_t *meta, ca
                     }
 
                     string_t inhex;
+                    string_init(inhex);
```

Comment:
This is a good catch!

Reply:
Thanks! Was wondering why the app was segfaulting :)